Tests/.pytest_cache/README.md

```diff
@@ -0,0 +1,8 @@
+# pytest cache directory #
```

Comment:
.pytest_cache directory need to be excluded from git repo. (add it to .gitignore)